src/databricks/labs/ucx/source_code/notebooks/sources.py

```diff
@@ -58,11 +61,13 @@ def to_migrated_code(self):
             sources.append('')  # following join will append lf
         return '\n'.join(sources)
 
-    def build_dependency_graph(self, parent: DependencyGraph) -> None:
+    def build_dependency_graph(self, parent: DependencyGraph, path_lookup: PathLookup) -> None:
```

Comment:
```suggestion
    def build_dependency_graph(self, parent: DependencyGraph) -> None:
```
don't change the signature.

Reply:
it's required, see below